tests/Cucumber/CompatibilityTest.php

```diff
@@ -158,6 +160,15 @@ private function normaliseFeature(?FeatureNode $featureNode): ?FeatureNode
             return null;
         }
 
+        if ($featureNode->getDescription() !== null) {
+            // We currently handle whitespace in feature descriptions differently to cucumber
+            // https://github.com/Behat/Gherkin/issues/209
+            // We need to be able to ignore that difference so that we can still run cucumber tests that
+            // include a description but are covering other features.
+            $trimmedDescription = preg_replace('/^\s+/m', '', $featureNode->getDescription());
```

Comment:
Hey, if we have to fix this here, doesn't that mean we should have just removed the counter hack from [CucumberNDJsonAstLoader](https://github.com/Behat/Gherkin/blob/e07986ef3e9b6621dcd7069831d96fef9c76d4ac/src/Loader/CucumberNDJsonAstLoader.php#L54)?

Reply:
Urgh, I missed that - the test implies that all the normalization happens here.

I agree, we should remove that hack from the NDJSON loader so it returns the exact cucumber parse result

Reply:
Fixed, thanks for pointing that out.